powerauth-java-crypto/src/main/java/io/getlime/security/powerauth/crypto/lib/model/ActivationStatusBlobInfo.java

```diff
@@ -150,18 +151,34 @@ public void setCtrLookAhead(byte ctrLookAhead) {
     }
 
     /**
-     * Get counter data.
-     * @return Counter data.
+     * Get least significant byte from the counter.
+     * @return Least significant byte from the counter.
      */
-    public byte[] getCtrData() {
-        return ctrData;
+    public byte getCtrInfo() {
```

Comment:
I do not like the `info` word in the method name, it is quite generic. Perhaps we can use something like `getCtrMasked` or `getCtrByte`... maybe you have a better idea.

Reply:
Renamed to `ctrByte` (e.g. `getCtrByte`).